nebari/stages/input_vars.py

```diff
@@ -125,7 +125,7 @@ def stage_02_infrastructure(stage_outputs, config):
             "kubeconfig_filename": os.path.join(
                 tempfile.gettempdir(), "NEBARI_KUBECONFIG"
             ),
-            **config.get("aws", {}).get("terraform_overrides", {}),
+            **config.get("amazon_web_services", {}).get("terraform_overrides", {}),
```

Comment:
These terraform overrides were not usable before as far as I can tell.  You'd get an error from pydantic that extra fields (the `aws` field) weren't allowed.

Reply:
Now they can be used via (e.g.)
```
amazon_web_services:
  terraform_overrides:
    existing_subnet_ids: ["subnet-05b2b1f41f0b1d8a6", "subnet-017efc3309fbca2da"]
    existing_security_group_id: "sg-0e2c865bdd8824b1a"
  region: ...
  <other_attributes>: ...
```

Reply:
Is it worth formalizing these the available `terraform_overrides` in the schema?